adventurine.rb

```diff
@@ -1,26 +1,87 @@
 require 'pp' #so the board prints prettier
 
 board = [
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
-  ["_","_","_","_","_","_",],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"],
+  ["_","_","_","_","_","_"]
 ]
 
-user_pos = [0,0]
-a = [-1,0]
-s = [0,1]
-w = [0,-1]
-d = [1,0]
+user_x = 0 
```

Comment:
I switched these from an array to individual variables because it made the code easier to read. It required too much brainpower for me to decipher `board[user_pos[0]][user_pos[1]]`